tests/test_plainhttpserver.py

```diff
@@ -0,0 +1,123 @@
+import unittest
+import logging
+import requests
+import concurrent.futures
+import json_logging
+from http.server import BaseHTTPRequestHandler
+from http.server import HTTPServer
+from typing import Any
+from json import loads
+from io import StringIO
+
+
+def setup_b3_logging(httpd: HTTPServer, level=logging.INFO):
+    # enable b3-propagation
+    # https://github.com/thangbn/json-logging-python#3-configuration
+    json_logging.init_plain_http_server(enable_json=True)
+    json_logging.init_request_instrument(httpd)
+
+    logging.basicConfig(level=level)
+    json_logging.config_root_logger()
+    logging.getLogger("plain-http-request-logger").setLevel(logging.WARNING)
+
+
+class RequestHandler(BaseHTTPRequestHandler):
```

Comment:
This should be part of the library itself. I think the idea will be user need to extend this custom requesthandler in order to extract request/response information.

Reply:
I agree